### PR TITLE
Create a prague file.

### LINKED ISSUE #31: vavrtom recommendations for Prague, Czech Republic
# Prague, Czech Republic

- [ ] Create a branch
- [ ] Add a file
- [ ] Commit my changes
- [ ] Open a pull request
- [ ] Collaborate with my friends
- [ ] Get a +1
- [ ] Merge the Pull Request to master

Anything I forgot, @krasmar?

### PR DESCRIPTION
**Pull request by vavrtom**
This PR closes #31 adding a file about Prague.

### CODE REVIEW

File: vavrtom.hometown.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## Prague
+
+### Places to eat:
```
Comment (vavrtom) [inline vavrtom.hometown.md]: Perfect!

File: vavrtom.hometown.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+4ta cenova
+
+### Fun things to do:
+
+well?
```
Comment (krasmar) [inline vavrtom.hometown.md]: well done

**Comment by adegoodyer** [comment]
@vavrtom Looks good :100:

File: vavrtom.hometown.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+## Prague
```
Comment (vavrtom) [inline vavrtom.hometown.md]: fdghfdgh

**Comment by stoe** [comment]
👏  looks great.

Please merge. 👍

**Pull request merged**